Extensions/PrimitiveDrawing/shapepainterruntimeobject.ts

```diff
@@ -274,6 +274,22 @@ namespace gdjs {
       return this._clearBetweenFrames;
     }
 
+    setCoordinatesAbsolute(value): void {
+      this._absoluteCoordinates = value;
+    }
+
+    setCoordinatesRelative(value): void {
+      this._absoluteCoordinates = !value;
+    }
+
+    areCoordinatesAbsolute(): boolean {
+      return this._absoluteCoordinates;
+    }
+
+    areCoordinatesRelative(): boolean {
+      return !this._absoluteCoordinates;
+    }
```

Comment:
Let's keep the interface simple by just having a single pair of these (the same one as the conditions/actions, for consistency).